PyInstaller/hooks/hook-queue.py

```diff
@@ -0,0 +1,22 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2016, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+"""
+Problem appears to be that pyinstaller cannot have two modules of the same
+name that differ only by lower/upper case.  The from the future 'queue' simply
+imports all of the 'Queue' module.  So, here we force the 'Queue' module to be
+imported.
+"""
+
+from PyInstaller.compat import is_py2
+
+if is_py2:
+    # force import of Python 2.7 Queue module
+    hiddenimports = ['Queue']
```

Comment:
I meant: This hook is called when a module `queue` is imported, and I assume `queue` is the module from the Python3 stdlib. And the Py3 stdlib for sure does not have an hidden import for the Python-2-module `Queue`.

But now I see that is a hack to make the package `future` https://pypi.python.org/pypi/future work. Please state this in the comments, since "future" is not know to all.